local-cluster/src/Hydra/LogFilter.hs

```diff
@@ -0,0 +1,82 @@
+-- | Utility functions to filter and simplify raw logs.
+module Hydra.LogFilter where
+
+import Control.Lens (Traversal', at, (.~), (?~), (^..), (^?))
+import Data.Aeson (Value (Array, Null))
+import Data.Aeson.Lens (key, values, _Object)
+import Hydra.Prelude
+
+filterLog :: Value -> Maybe Value
+filterLog entry = do
+  guard (entry ^? key "message" . key "tag" == Just "Node")
+  node <- entry ^? key "message" . key "node"
+  result <- case node ^? key "tag" of
```

Comment:
So you mean I can throw away my jq queries :smile: ?

```
jq '.message.node?|select(.tag? == "ProcessedEffect")|.effect?.serverOutput?|select(.tag == "SnapshotConfirmed")|.snapshot.confirmedTransactions|map(.id)
```

Reply:
That works for _one_ message, but what about the others ;) ?